product/rider_feedback/discovery.md

```diff
@@ -0,0 +1,34 @@
+### Overview
+
+An important component of the SMARTA and Ataper projects is the eventual incorporation of user-generated feedback as an organic, discrete source of system events, behaviors, alerts, and foibles. This data source would service the needs of ostensibly ephemeral impact events for the system and ultimately feed a data warehouse for BI analysis. 
+
+The conceit of the story is that primary sources of truth are fundamentally invalidated or otherwise eroded by one-off events that are unaccounted for in the existing API.
+
+### Goals
+
+Our exaxmple for the first iteration is a riff on the Waze-style user feedback system with the key characteristics:
+
+* Pre-defined event categories.
+* A confirm/deny system for each event to invalidate out-of-date/resolved incidents.
+* Pre-emptive visual representation at impacted stations/lines.
+* Traceability to individual rides and riders. 
```

Comment:
I see that we have a `Persistence and Identity` section, but I think this will definitely be an interesting conversation, at least based on the last meeting we had.  I'm curious to see whether this is something we should design first for, or if it should be something that will have add-on value.

Reply:
While I definitely think persistence and identity (and traceability to individual rides) are important for the feedback system and make enable a whole world of use cases/improvements, I also think that the core functionality or providing arrival estimates doesn't require us to create any server-side records about a user other than those used for API authentication, and I'd personally favor giving users the option of keeping their data footprint that small if we can

Reply:
That said, I think providing user feedback should entail that we're going to create some sort of identity for you, since it'll be really important for detecting fraud and pruning out bullshit feedback!